sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/samples/Sample1_AddAndRemoveRoleAssignment.cs

```diff
@@ -23,36 +23,24 @@ public void AddAndRemoveRoleAssignmentSync()
             // Environment variable with the Synapse workspace endpoint.
             string workspaceUrl = TestEnvironment.WorkspaceUrl;
 
-            #region Snippet:AccessControlSample1AccessControlClient
             AccessControlClient client = new AccessControlClient(new Uri(workspaceUrl), new DefaultAzureCredential());
-            #endregion
 
-            #region Snippet:AccessControlSample1GetWorkspaceAdminRole
             SynapseRole role = client.GetRoleDefinitions().Single(role => role.Name == "Workspace Admin");
```

Comment:
I wonder if it will be easier for a user to digest this better if it is more divided.
```
Response<...> response = client.GetRoleDefinitions();
SynapseRole role = response.Value.Single(role => role.Name == "Workspace Admin");
```
It will allow them to see the return types and explore that too.